tests/unit/schema/models/schemaManger.spec.ts

```diff
@@ -16,9 +16,23 @@ const schemas: Schema[] = [
 
 describe('SchemaManager', () => {
   let schemaManager: SchemaManager;
-  beforeAll(() => {
-    schemaManager = new SchemaManager(schemas);
+  let getField: jest.Mock;
+  let getFields: jest.Mock;
+  let getDomainFieldsList: jest.Mock;
+
+  beforeEach(() => {
+    getField = jest.fn();
+    getFields = jest.fn();
+    getDomainFieldsList = jest.fn();
+    schemaManager = new SchemaManager(schemas, { getField, getFields, getDomainFieldsList });
+  });
+
+  afterEach(() => {
+    getField.mockClear();
+    getFields.mockClear();
+    getDomainFieldsList.mockClear();
```

Comment:
there's a jest function to clear all mocks, not sure about the name.

Reply:
done